### PR TITLE
Added User Profile Page

### PR DESCRIPTION
**Pull request by Ankur9669**
image

### CODE REVIEW

**Review by varun1233 — APPROVED** [review]
Nice Work. Code looks good to merge

File: userprofile.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+.user-profile-container{
+    display: grid;
+    grid-template-columns: 14.3rem 1fr;
+    margin: 1rem 0 0 0;
+}
+.user-info-container{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    max-width: 28.1rem;
```
Comment (sri-rishi) [inline userprofile.css:41]: Suggestion: can use only margin-top: 1rem as it has only one top property

**Review by sri-rishi — CHANGES_REQUESTED** [review]
Great work, just one suggestion which I mentioned will better the code

File: userprofile.css (hunk 1 above)
Comment (Ankur9669) [reply userprofile.css:41]: Done

**Pull request merged**